src/tools/tidy/src/main.rs

```diff
@@ -19,6 +19,11 @@ use std::path::{PathBuf, Path};
 use std::env;
 
 macro_rules! t {
+    ($e:expr, $p:expr) => (match $e {
+        Ok(e) => e,
+        Err(e) => panic!("{} failed on {} with {}", stringify!($e), ($p).display(), e),
+    });
```

Comment:
While I agree that it's nice to have a path name in error messages, it was somewhat intentional that this doesn't diverge from `try!` (as this shows up elsewhere). It doesn't really matter too much though.